.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Release
+
+on:
+  schedule:
+    - cron: '0 11 * * 1' # Run every Monday at 7am NYC time
+      branches:
+        - master
+  workflow_dispatch: # allows manual releasing
+      branches:
+        - master
+jobs:
+  build_and_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
nit: there is a v4 - since this is a new workflow you probably want the latest of the deps